keras/layers/convolutional.py

```diff
@@ -373,6 +373,9 @@ class Conv2D(_Conv):
             Specifying any stride value != 1 is incompatible with specifying
             any `dilation_rate` value != 1.
         padding: one of `"valid"` or `"same"` (case-insensitive).
+            Note that `"same"` is inconsistent across backends with `strides` != 1,
+            as described at
+            https://github.com/keras-team/keras/pull/9473#issuecomment-372166860
```

Comment:
Please format the link ("as described [here](...)")

Reply:
Also say "may be slightly inconsistent"

since 1) it is not necessarily the case depending on various parameters, 2) the difference is minor and generally unimportant